hs_core/receivers.py

```diff
@@ -141,3 +146,24 @@ def metadata_element_pre_update_handler(sender, **kwargs):
         return {'is_valid': True, 'element_data_dict': element_form.cleaned_data}
     else:
         return {'is_valid': False, 'element_data_dict': None, "errors": element_form.errors}
+
+
+@receiver(post_add_generic_aggregation)
+@receiver(post_add_geofeature_aggregation)
+@receiver(post_add_raster_aggregation)
+@receiver(post_add_netcdf_aggregation)
+@receiver(post_add_timeseries_aggregation)
+@receiver(post_add_reftimeseries_aggregation)
+@receiver(post_remove_file_aggregation)
+@receiver(post_delete_resource)
+@receiver(post_raccess_change)
+def hs_update_web_services(sender, **kwargs):
+    """Signal to update resource web services."""
+
+    update_web_services.apply_async((
+        settings.HSWS_URL,
+        settings.HSWS_API_TOKEN,
+        settings.HSWS_TIMEOUT,
+        settings.HSWS_PUBLISH_URLS,
+        kwargs.get("resource").short_id
```

Comment:
@kjlippold probably it is better to pass in resource short_id directly to update_web_services task rather than pass in resource object, then retrieve short_id from resource object, if all that update_web_services needs is the short_id.

Reply:
The new signals all provide the resource object, and the receiever gets that object and then passes the short_id of the resource to the update_web_services_task. Most of the other signals seem to include the resource object, not just the short_id, so I tried to follow that pattern with the new signals, but only the short id is being passed to the task.

Reply:
@kjlippold In that case, I think it'd be cleaner and better for signals just to provide the resource short id for receivers to use if the short id is all receivers need. This is a minor comment, so I'll leave it up to you whether you want to improve it or not. @sblack-usu and/or @cuahsimarko do you also want to take a final look at this PR?